addons/osfstorage/utils.py

```diff
@@ -43,7 +43,7 @@ def update_analytics(node, file, version_idx, session_key, action='download'):
     node_info = {
         'contributors': contributors
     }
-    resource = node.guids.first()
+    resource = node.get_guid()
 
     update_counter(resource, file, version=None, action=action, node_info=node_info, session_key=session_key)
     update_counter(resource, file, version_idx, action, node_info=node_info, session_key=session_key)
```

Comment:
So, if we provide a preprint version, should this `update_counter` handle the version instead of the base guid only? Each preprint version has different file. Please take a look to check download count for each preprint version.

Please note that the `version=None` and `version_idx` is the file version and we don't need to worry about it.

This might be a new scope that needs to be added to the project requirement but take a look first at how `update_counter`works.

Reply:
Per check-in/scrum discussion, this will be a separate ticket to look into the counter. Download count seems fine with fix from this PR.